libipl/p10/common.C

```diff
@@ -196,3 +198,22 @@ bool ipl_check_functional_master(void)
 
 	return true;
 }
+
+void  log_sbe_ffdc(struct pdbg_target *pib)
```

Comment:
Please use `ipl_` prefix for public functions.

Reply:
Ack